src/network/networkd-network.h

```diff
@@ -84,6 +84,15 @@ typedef enum RADVPrefixDelegation {
         _RADV_PREFIX_DELEGATION_INVALID = -1,
 } RADVPrefixDelegation;
 
+typedef enum MultiCastRouter {
+        MULTICAST_ROUTER_NONE,
+        MULTICAST_ROUTER_TEMPORARY_QUERY,
+        MULTICAST_ROUTER_PERMANENT,
+        MULTICAST_ROUTER_TEMPORARY,
```

Comment:
The values are defined in `if_bridge.h`. So, how about the following.
```
typedef enum MulticastRouter {
  MULTICAST_ROUTER_NONE = MDB_RTR_TYPE_DISABLED,
  MULTICAST_ROUTER_TEMPORARY_QUERY = MDB_RTR_TYPE_TEMP_QUERY,
  MULTICAST_ROUTER_PERMANENT = MDB_RTR_TYPE_PERM,
  MULTICAST_ROUTER_TEMPORARY = MDB_RTR_TYPE_TEMP,
} MulticastRouter;
```

Reply:
yes but that throws errors 
```In file included from ../src/basic/linux/if_bridge.h:19,
                 from ../src/network/networkd-network.h:24,
                 from ../src/network/networkd-manager.h:19,
                 from ../src/network/networkd-dhcp6.c:17:
../src/basic/linux/in6.h:33:8: error: redefinition of ‘struct in6_addr’
   33 | struct in6_addr {
      |        ^~~~~~~~
In file included from ../src/systemd/sd-radv.h:24,
                 from ../src/network/networkd-dhcp6.c:8:
/usr/include/netinet/in.h:211:8: note: originally defined here
  211 | struct in6_addr
      |        ^~~~~~~~
In file included from ../src/basic/linux/if_bridge.h:19,
                 from ../src/network/networkd-network.h:24,
                 from ../src/network/networkd-manager.h:19,
                 from ../src/network/networkd-dhcp6.c:17:
../src/basic/linux/in6.h:41: warning: "s6_addr" redefined
   41 | #define s6_addr   in6_u.u6_addr8
      | 
In file included from ../src/systemd/sd-radv.h:24,
                 from ../src/network/networkd-dhcp6.c:8:
/usr/include/netinet/in.h:219: note: this is the location of the previous definition
  219 | #define s6_addr   __in6_u.__u6_addr8
      | 
In file included from ../src/basic/linux/if_bridge.h:19,
                 from ../src/network/networkd-network.h:24,
                 from ../src/network/networkd-manager.h:19,
                 from ../src/network/networkd-dhcp6.c:17:
../src/basic/linux/in6.h:43: warning: "s6_addr16" redefined
   43 | #define s6_addr16  in6_u.u6_addr16


```